doc/policies/webui.rst

```diff
@@ -300,6 +300,22 @@ being disabled.
 
 (Since privacyIDEA 3.0)
 
+token_rollover
+~~~~~~~~~~~~~~
+
+type: str
+
+This is a whitespace separated list of tokentypes, for which a rollover button is
+displayed in the token details and in the token list. This button will generate a
+new token secret for the displayed token.
+
+This e.g. enables a user to transfer a softtoken to a new device while keeping the
+token number restricted to 1.
+
+This policy does not work for all token types. Use it with totp, hotp, push, paper or tan.
+
+(Since privacyIDEA 3.6)
```

Comment:
We decided that to be a webui policy. Since enrollment policies also work during enrollment on the API level.
This however, is a pure UI policy, since it displays a button, that leads to an API functionality, that works anyways.